styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+nav {
+  padding: 1.2rem;
+}
+
+ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.welcome {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 2rem;
+  line-height: 1.5;
+  color: #28352f;
+}
+
+.header-section-title {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  text-align: center;
+}
```

Comment:
- [ ] :thinking: In my opinion, to stick to the Figma design it would be better to use the same font family for the title and all the other parts . (font-family: 'Inter';)

Reply:
- Just as @GedeonTS has suggested, kindly stick to the font family on Figma 🤗